### ISSUE TITLE
hook returns gray image

### ISSUE THREAD

**Issue by nedimcimpo** [ivansevillaa/use-next-blurhash#14]
When using hook url returned is always gray image

<img width="449" alt="Screenshot 2023-02-20 at 19 00 32" src="https://user-images.githubusercontent.com/10185705/220174780-7833b6a8-2d92-4a1b-9fcb-9b186f96e34d.png">

When using same hash on blurga.sh this is what I get, so hash is ok

<img width="858" alt="Screenshot 2023-02-20 at 19 00 46" src="https://user-images.githubusercontent.com/10185705/220174917-4585cc88-6085-46f5-8cc4-ea998ef597a0.png">

And the image I'm trying to hash
![Uploading Screenshot 2023-02-20 at 19.00.53.png…]()

**Comment by Weijtmans** [comment]
I have the same issue

**Comment by andyszy** [comment]
I have this issue also

**Comment by tavaneftekhar** [comment]
Same issue here.

**Comment by duveborg** [comment]
Same

**Comment by mauriciabad** [comment]
LOL, the most basic feature is broken for almost a year

**Comment by ivansevillaa** [comment]
yes... sorry for that I have this a little abandoned. meanwhile you can use previous version that works well

**Comment by tavaneftekhar** [comment]
which previous version are you referring to? 0.1.0?

**Comment by Stevemoretz** [comment]
> yes... sorry for that I have this a little abandoned. meanwhile you can use previous version that works well

Went a few versions back and none of them work all give the same error:

```
./node_modules/canvas/build/Release/canvas.node
Module parse failed: Unexpected character '�' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
(Source code omitted for this binary file)
```

**Comment by Stevemoretz** [comment]
Hi everyone I wrote a replacement check out:
https://github.com/ivansevillaa/use-next-blurhash/issues/15
